src/Tasks/Common/Resources/xlf/Strings.cs.xlf

```diff
@@ -1,354 +1,354 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
UTF-8 BOM removed. It is not required so I'll let that go, but noting the diff.

Reply:
If possible, consider changing the tooling that generated this to leave the BOM/NO-BOM state intact.